### DISCUSSION TITLE
Is it possible to run Stride Game Engine with Wine?

### DISCUSSION THREAD

**Discussion by GATETECAT4** [stride3d/stride#1993] [Q&A]
Since Stride still doesn't have a Linux version that I know of, I came up with the idea of running it with Wine, but I don't know if it works well or not, so I'd like to know if it's possible.

Thank you translator Deepl :u.

**Comment by Doprez** [comment]
Some people have tried with WINE and as far as I know they havent had success. I think @JeromyWalsh had it running in a VM for Linux but this was a part of some discussions in the Linux channel in the Discord.

**Comment by NicusorN5** [comment]
I was able to run Stride games with Wine flawlessly, but not the editor.

**Reply by Doprez** [reply]
Oh good point, published games seem to run fine on Linux. I tried with Mint and OpenGL with no issues.
